generators/generator-adaptive-bot/generators/app/templates/dotnet/functions/Startup.cs

```diff
@@ -11,21 +14,18 @@ public class Startup : FunctionsStartup
         public override void Configure(IFunctionsHostBuilder builder)
         {
             builder.Services.AddBotRuntime(builder.GetContext().Configuration);
+
+            // The workaround below will be removed next SDK patch, when this bug fix gets released: https://github.com/microsoft/botbuilder-dotnet/issues/5239
+            // In the meantime, we're guaranteed to have CoreAdapter registered as IBotFrameworkHttpAdapter, so look it up and register it as BotAdapter.
+            builder.Services.AddSingleton(sp => (BotAdapter)sp.GetServices<IBotFrameworkHttpAdapter>().First(a => typeof(BotAdapter).IsAssignableFrom(a.GetType())));
```

Comment:
I hate this! If there is an Rc1 it will go away, otherwise we'll ship a 4.12.1 right after 4.12 to remove this line.